framework/src/org/apache/cordova/CordovaActivity.java

```diff
@@ -95,6 +95,9 @@ Licensed to the Apache Software Foundation (ASF) under one
     protected ArrayList<PluginEntry> pluginEntries;
     protected CordovaInterfaceImpl cordovaInterface;
 
+    //flag for case when appView is null during onStart
+    private boolean shouldHandleStartOnAppViewInit = false;
```

Comment:
The webview (and thus plugins) are generally created from onCreate(), which means they will be initialized before the first call to onStart. If you're unit tests passes without this flag (and without the one in CordovaWebViewImpl), then I think you should remove them. It's not obvious (to me anyways) that this behaviour would even be the desired behaviour. If there's a reason it needs to be this way, then let me know! But if so, then we should upated onResume as well so that we're consistent.